src/sentry/workflow_engine/handlers/condition/event_frequency_handlers.py

```diff
@@ -0,0 +1,70 @@
+from collections import defaultdict
+from datetime import datetime, timedelta
+from typing import Any
+
+from sentry import tsdb
+from sentry.issues.constants import get_issue_tsdb_group_model
+from sentry.models.group import Group
+from sentry.rules.conditions.event_frequency import STANDARD_INTERVALS, percent_increase
+from sentry.tsdb.base import TSDBModel
+from sentry.workflow_engine.handlers.condition.event_frequency_base_handler import (
+    BaseEventFrequencyConditionHandler,
+)
+from sentry.workflow_engine.models.data_condition import Condition
+from sentry.workflow_engine.registry import condition_handler_registry
+from sentry.workflow_engine.types import DataConditionHandler, DataConditionResult
+
+
+class EventFrequencyConditionHandler(BaseEventFrequencyConditionHandler):
```

Comment:
nit: if we're going to have all these classes in 1 file, might as well include the base here too (otherwise, i'd split each of these classes as a separate file, but that's a bit JS-y too)

Reply:
there are 4 existing condition classes that are related but represent 2 conditions (count and percent change), and i am group together these 2 very similar conditions